test/MinterGateway.t.sol

```diff
@@ -2940,4 +2983,243 @@ contract MinterGatewayTests is TestUtils {
         vm.expectRevert();
         minterGateway_.mintM(3);
     }
+
+    /* ============ Sherlock ============ */
+
+    function test_maliciousValidator_signatureReuse() external {
```

Comment:
I don't see any signature reuse in this test. Both tests seem to test the same scenario, one of the validators, the malicious one, provides a signature older than the proposed retrieval.

Reply:
I am removing these tests since they are no longer possible due to the fix.the stale update tests cover these